### PR TITLE
Update GitHub extension

### PR DESCRIPTION
**Pull request by xilopaint**
## Description

I believe emojis cheapen the design, so I swapped them with Raycast's built-in icons for showing PR statuses. Placing icons before the PR titles enhances readability and highlights the statuses.

## Screenshot

Screenshot 2024-02-25 at 23 28 37
Screenshot 2024-02-26 at 02 37 26

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/github/CHANGELOG.md
Comment (xilopaint) [reply extensions/github/CHANGELOG.md]: Done.

File: extensions/github/src/open-pull-request-menu.tsx
Comment (xilopaint) [reply extensions/github/src/open-pull-request-menu.tsx:42]: Are you sure the colors make it look better and easier to scan? I personally find that they make it look a bit messy:

Screenshot 2024-02-26 at 07 47 44

Screenshot 2024-02-26 at 07 50 01

Colors in the menu bar hardly work well, it's no wonder that Apple rarely uses them. On the other hand, I'd agree it looks good In Raycast since the UI is more complex:

Screenshot 2024-02-26 at 07 57 50

But that's just my opinion. Tell me your final word.

Comment (nesl247) [reply extensions/github/src/open-pull-request-menu.tsx:42]: I think it makes it look a lot better. I actually think Apple does it wrong by not using colors. But this is just my own personal opinion.

Comment (xilopaint) [reply extensions/github/src/open-pull-request-menu.tsx:42]: It's impossible to maintain design consistency when using a bunch of colors in the menu bar. Colors can make the menu bar distracting, so you should only focus on it when necessary. Apple uses colors in the menu bar sparingly as shown in this example:

Screenshot 2024-02-26 at 12 49 56

Comment (xilopaint) [reply extensions/github/src/open-pull-request-menu.tsx:42]: Distraction is helpful when it focuses your attention where needed. For instance, the "Unread Notification" menu bar command turns orange, standing out from other icons and signaling a new GitHub notification.

In our case, you don't need to stress the UI by using colors because the icons are sufficient to differentiate between the statuses. By doing so, you create a visual hierarchy that directs your attention first to the icon and only then to the text, which is what matters in the end. It's no wonder that SF Symbols are so commonly used in menu bar apps. They are designed to integrate seamlessly with the San Francisco system font conveying information graphically without visual distractions.

Comment (nesl247) [reply extensions/github/src/open-pull-request-menu.tsx:42]: Maybe just make it a setting?

Comment (qeude) [reply extensions/github/src/open-pull-request-menu.tsx:42]: I agree with @nesl247, as it seems to be more a personal preference thing, we might want to make a setting out of it.

Comment (thomaslombart) [reply extensions/github/src/open-pull-request-menu.tsx:42]: As you want, I'm not going to block the PR anyway. I'm fine shipping it without colors 🙂

**Review by thomaslombart — APPROVED** [review]

**Pull request merged**
